app/components/ContentBlocks/TrackingGA.tsx

```diff
@@ -0,0 +1,11 @@
+import { useEffect } from "react";
+import { trackSiteVisit } from "~/utils/gtag.client";
+import type { ITrackingGaFields } from "../../../@types/generated/contentful";
+
+export default function TrackingGa({ title }: ITrackingGaFields) {
+  useEffect(() => {
+    trackSiteVisit({ label: title });
+  });
```

Comment:
I‘d pass an empty array here so that the effect runs once. Re-renders should not happen in our setup, but still: Better safe than sorry 😅

Reply:
Right, thanks for spotting it! Just added it